### PR TITLE
Homework week 2 Danijel Cvitkusic

### PR DESCRIPTION
**Pull request by DanijelCvit**

### CODE REVIEW

File: 3-UsingAPIs/Week2/homework/ex5-vscDebug.js [hunk 1]
```diff
@@ -23,12 +25,13 @@ function renderLaureates(laureates) {
...
 async function fetchAndRender() {
   try {
-    const laureates = getData(
+    const { laureates } = await getData(
       'http://api.nobelprize.org/2.0/laureates?birthCountry=Netherlands&format=json&csvLang=en'
     );
     renderLaureates(laureates);
   } catch (err) {
     console.error(`Something went wrong: ${err.message}`);
+    console.log(err.stack);
```
Comment (jontra000) [inline 3-UsingAPIs/Week2/homework/ex5-vscDebug.js:34]: Don't forget to delete the stack log!

**Review by jontra000 — CHANGES_REQUESTED** [review]
Looks good @DanijelCvit. Just the bug in ex2 to fix and then I can approve.

File: 3-UsingAPIs/Week2/homework/ex1-programmerFun/index.js [hunk 2]
```diff
@@ -18,28 +18,39 @@ Full description at: https://github.com/HackYourFuture/Homework/blob/main/3-Usin
...
-  // TODO return a promise using `fetch()`
+  return fetch(url)
+    .then((response) => {
+      if (response.ok) {
+        return response.json();
+      } else {
+        throw 'HTTP ERROR';
+      }
+    })
+    .then((jsonData) => {
```
[the remark this answers is not in the document]
Comment (DanijelCvit) [reply 3-UsingAPIs/Week2/homework/ex1-programmerFun/index.js]: Thanks.Removed.

File: 3-UsingAPIs/Week2/homework/ex2-pokemonApp/index.js [hunk 3]
```diff
@@ -22,18 +22,67 @@ Use async/await and try/catch to handle promises.
 Try and avoid using global variables. As much as possible, try and use function 
 parameters and return values to pass data back and forth.
 ------------------------------------------------------------------------------*/
-function fetchData(/* TODO parameter(s) go here */) {
```
[the remark this answers is not in the document]
Comment (DanijelCvit) [reply 3-UsingAPIs/Week2/homework/ex2-pokemonApp/index.js:25]: Ah yes. Fixed it :)

**Comment by jontra000** [comment]
Looks good! Approved.

**Pull request closed without merging**